repose-aggregator/tests/performance-tests/roles/gatling/files/simulations/filters/regexrbac/RegexRbacFilterSimulation.scala

```diff
@@ -0,0 +1,120 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+
+package filters.regexrbac
+
+import com.typesafe.config.ConfigFactory
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import io.gatling.http.request.builder.HttpRequestBuilder
+
+import filters.regexrbac.RegexRbacFilterSimulation._
+
+import scala.concurrent.duration._
+
+/**
+  * RegEx RBAC filter performance simulation.
+  */
+class RegexRbacFilterSimulation extends Simulation {
+  // properties to configure the Gatling test
+  val conf = ConfigFactory.load("application.conf")
```

Comment:
Is this boilerplate? If so we should totally create our own subclass of `Simulation` to handle this crap.

Reply:
It sure kind of looks like it. I will defer to the original creators of the performance test suite, @Mario-Lopez and @dmnjohns, as to why or if this is needed.

Reply:
It is boilerplate. I mean, you can use these values to configure your simulation, but I would view them as a reasonable suggestion. I've lost enough familiarity with Gatling that I could not say, without looking into Gatling again, whether or not creating an inheritable `Simulation` is a good idea. In general, doing so increases code reuse but (potentially) reduces flexibility. Not sure which one matters more for these tests.

Reply:
I don't currently see us changing these values much, and a lot of this boiler plate is loading up settings from elsewhere, we would probably just tweak those settings in the elsewhere and this code wouldn't change

Reply:
This is outside the current scope and has been opened as:

* https://repose.atlassian.net/browse/REP-6336